pkg/crc/preflight/preflight_checks_common.go

```diff
@@ -110,17 +110,6 @@ func fixBundleExtracted(bundlePath string, preset crcpreset.Preset) func() error
 		if err := os.MkdirAll(bundleDir, 0775); err != nil {
 			return fmt.Errorf("Cannot create directory %s: %v", bundleDir, err)
 		}
-		if err := validation.ValidateBundle(bundlePath, preset); err != nil {
-			var e *validation.InvalidPath
-			if !errors.As(err, &e) {
-				return err
-			}
```

Comment:
This may be slightly better than what ValidateBundlePath is doing though, this does not ignore all errors, just invalid paths.

Reply:
But ignoring the invalid path at this step doesn't make sense, this function suppose to fix the bundle extraction and all the errors are handled even invalid path one as part of validation run of different command.